packages/@aws-cdk/cfnspec/build-tools/create-missing-libraries.ts

```diff
@@ -0,0 +1,247 @@
+              artifactId: javaArtifactId
+            }
+          },
+          sphinx: {}
+        }
+      },
+      repository: {
+        type: "git",
+        url: "https://github.com/awslabs/aws-cdk.git"
+      },
+      homepage: "https://github.com/awslabs/aws-cdk",
+      scripts: {
+        build: "cdk-build",
+        integ: "cdk-integ",
+        lint: "cdk-lint",
+        package: "cdk-package",
+        pkglint: "pkglint -f",
+        test: "cdk-test",
+        watch: "cdk-watch"
+      },
+      'cdk-build': {
+        cloudformation: namespace
+      },
+      keywords: [
+        "aws",
+        "cdk",
+        "constructs",
+        namespace,
+        moduleName
+      ],
+      author: {
+        name: "Amazon Web Services",
+        url: "https://aws.amazon.com",
+        organization: true
+      },
+      license: "Apache-2.0",
+      devDependencies: {
+        "@aws-cdk/assert": `^${version}`,
+        "cdk-build-tools": `^${version}`,
+        "cfn2ts": `^${version}`,
+        "pkglint": `^${version}`,
+      },
+      dependencies: {
+        "@aws-cdk/cdk": `^${version}`,
+      },
+      peerDependencies: {
+        "@aws-cdk/cdk": `^${version}`,
+      },
+      engines: {
+        node: '>= 8.10.0'
+      }
+    });
+
+    await write('.gitignore', [
+      '*.d.ts',
+      '*.generated.ts',
+      '*.js',
+      '*.js.map',
+      '*.snk',
+      '.jsii',
+      '.LAST_BUILD',
+      '.LAST_PACKAGE',
+      '.nycrc',
+      '.nyc_output',
+      'coverage',
+      'dist',
+      'tsconfig.json',
+      'tslint.json',
+    ]);
+
+    await write('.npmignore', [
+      '# The basics',
+      '*.ts',
+      '*.tgz',
+      '*.snk',
+      '!*.d.ts',
+      '!*.js',
+      '',
+      '# Coverage',
+      'coverage',
+      '.nyc_output',
+      '.nycrc',
+      '',
+      '# Build gear',
+      'dist',
+      '.LAST_BUILD',
+      '.LAST_PACKAGE',
+      '.jsii',
+    ]);
+
+    await write('lib/index.ts', [
+      `// ${namespace} CloudFormation Resources:`,
+      `export * from './${lowcaseModuleName}.generated';`
+    ]);
+
+    await write(`test/test.${lowcaseModuleName}.ts`, [
+      "import { Test, testCase } from 'nodeunit';",
+      "import {} from '../lib';",
+      "",
+      "export = testCase({",
+      "    notTested(test: Test) {",
+      "        test.ok(true, 'No tests are specified for this package.');",
+      "        test.done();",
+      "    }",
+      "});",
+    ]);
+
+    await write('README.md', [
+      `## ${namespace} Construct Library`,
+      '',
+      '```ts',
+      `import ${lowcaseModuleName} = require('${packageName}');`,
+      '```',
+    ]);
+
+    const templateDir = path.join(__dirname, 'template');
+    for (const file of await readdir(templateDir)) {
+      await copyFile(path.join(templateDir, file), path.join(pacakgePath, file));
+    }
+
+    // bootstrap and build the package and all deps to ensure integrity
+    await exec(`lerna bootstrap --scope ${packageName}`);
```

Comment:
Would get path to `lerna` from dependencies.

Reply:
Done.